week2/style.css

```diff
@@ -0,0 +1,101 @@
+.main {
+  display: flex;
+  position: absolute;
+  padding: 100px;
+  height: 200px;
+  width: 200px;
+  z-index: 2;
+}
+
+.text-box {
+  display: flex;
+  position: absolute;
+  padding: 100px;
+  height: 200px;
+  width: 200px;
+  z-index: 1;
+}
+
+h1 {
+  margin: auto;
+}
+
+/* Individual box transitions */
+#boxOne {
+  height: 100px;
```

Comment:
Good implementation could have been to create a separate class called .box ans add all the common CSS into it and reuse the same class in additions with custom classesz.

Reply:
Thanks, made the recommended changes!